cilium/datadog_checks/cilium/data/conf.yaml.example

```diff
@@ -28,14 +28,14 @@ instances:
     #
     # timeout: 10
 
-    ## @param metrics - list of strings - required
+    ## @param metrics - list of strings - optional
     ## List of metrics to be fetched from the prometheus endpoint, if there's a
-    ## value it'll be renamed. This list should contain at least one metric
+    ## value it'll be renamed.
     #
-    metrics:
-      - processor: cpu
-      - memory: mem
-      - io
+    # metrics:
+    #  - processor: cpu
+    #  - memory: mem
+    #  - io
```

Comment:
Relevant bit of code:

https://github.com/DataDog/integrations-core/blob/a88192c4a81c95421568d95f1cec3151095cab4c/cilium/datadog_checks/cilium/cilium.py#L39

✅